README.md

```diff
@@ -92,8 +121,172 @@ export class MyComponent implements OnInit {
 }
 ```
 
+#### Usage of Forms services
+- the `exampleFormService` service is a generated service holding a `FormGroup` definition that corresponds
+ with the request data structure
+
+```html
+<form [formGroup]="exampleFormService.form" (ngSubmit)="sendForm()" class="full-width">
+
+    <input type="text" name="email" required  placeholder="email"
+           [formControl]="exampleFormService.form.get('email')" />
+
+    <button type="submit"
+            [disabled]="exampleFormService.form.invalid">Save</button>
+</form>
+```
+
+- the generated service looks like this
+```typescript
+export class ExampleFormService {
+  form: FormGroup;
+  constructor(
+    private exampleService: ExampleService,
+  ) {
+    this.form = new FormGroup({
+      email: new FormControl(undefined, [Validators.email, Validators.required]),
+      });
+  }
+}
+```
+
+#### Usage of generated NGRX workflow
+
+##### Import the generated module
+- import the generated module to yours
+```typescript
+@NgModule({
+  imports: [
+    ...,
+    ExampleModule,
+    ...,
+  ],
+})
+export class YourModule {}
+```
+
+- The generated module looks like this:
+```typescript
+@NgModule({
+  imports: [
+    FormsSharedModule,
+    NgrxStoreModule.forFeature(selectorName, ExampleReducer),
+    NgrxEffectsModule.forFeature([ExampleEffects]),
+  ],
+  providers: [
+    ExampleService,
+    ExampleFormService,
+  ],
+})
+export class ExampleModule {}
+```
+
+##### Component (created by you)
+- In the component, send the above created form via `sendForm()` method. Notice the way a generated anction is dispatched.
+```typescript
+import {Component, OnDestroy} from '@angular/core';
+import {Store} from '@ngrx/store';
+import {takeUntil} from 'rxjs/operators';
+import {Subject} from 'rxjs/Subject';
+import {ExampleFormService} from '../../generated/store/example/exampleModule/example.service';
+import {Start as ExampleStart} from '../../generated/store/example/exampleModule/states/actions';
+import {AppState} from '../states/exmaple.models';
+
+@Component({
+  selector: 'example-component',
+  templateUrl: 'example-component.html',
+})
+export class ExampleComponent implements OnDestroy {
+
+  ngDestroy = new Subject<void>();
+
+  constructor(
+    public exampleFormService: ExampleFormService,
+    private store: Store<AppState>,
+  ) { }
+
+  sendForm() {
+    this.store.dispatch(new ExampleStart(this.exampleFormService.form.value));
+  }
+
+  ngOnDestroy() {
+    this.ngDestroy.next();
+    this.ngDestroy.complete();
+  }
+}
+```
+
+##### Effect (generated)
+- the dispatched method is intercepted by an effect which calls the API via the generated API service
```

Comment:
Let's make it a sentence (capital letter, period).

Reply:
it would be inconsistent with the rest